Makefile

```diff
@@ -9,9 +9,10 @@ ORG ?= rancher
 # last commit on 2021-10-06
 TAG ?= 441e06f11bfafcf5f818975298943537a103b5a8$(BUILD_META)
 
-ifneq ($(DRONE_TAG),)
-TAG := $(DRONE_TAG)
-endif
+# Temporarily disable this as Github tags can't be a SHA (too long)
+#ifneq ($(DRONE_TAG),)
+#TAG := $(DRONE_TAG)
+#endif
```

Comment:
Should we maybe set `DRONE_COMMIT_SHA` as a replacement?

https://docs.drone.io/pipeline/environment/reference/drone-commit-sha/